sormas-api/src/main/java/de/symeda/sormas/api/person/JournalPersonDto.java

```diff
@@ -33,6 +35,8 @@
 	private Sex sex;
 	@SensitiveData
 	private Date latestFollowUpEndDate;
+	@SensitiveData
```

Comment:
Normally enum and date fields are not sensitive, is this a special case?

Reply:
Well, I thought if the latestFollowUpEndDate is sensitive, the followUpStatus should be, too.
Is this tag used for pseudonomization? If so, maybe it's not a bad idea to leave it like that, because the JournalPersonDto is involved in transmission of person data to external systems.
I'm not sure about this, though.

Reply:
Yes `@SensitiveData` and `@PersonalData` annotations are used for pseudonymization 
For me it seems that `latestFollowUpEndDate` is wrongly marked as sensitive data.

The requirement for *personal data* and *sensitive data* is:  data from which somebody can identify a person is personal and all personal data and all free text and things relates to the location of the case/contact/person/sample/etc is sensitive.

I don't think it makes sense to pseudonymize `latestFollowUpEndDate` and `followUpStatus`.

Reply:
Okay, I removed both of the tags.